awesome_owl/static/src/card/card.js

```diff
@@ -0,0 +1,14 @@
+import { Component, useState } from "@odoo/owl";
+
+export class Card extends Component {
+    static template = "awesome_owl.card";
```

Comment:
nitpick: prefer PascalCase for template names
```suggestion
    static template = "awesome_owl.Card";
```